garden-backend-service/src/api/search/sql.sql

```diff
@@ -37,35 +52,62 @@ CREATE MATERIALIZED VIEW IF NOT EXISTS garden_documents AS
 
 
 CREATE MATERIALIZED VIEW IF NOT EXISTS entrypoint_documents AS
-    SELECT id,
+    SELECT e.id,
     setweight(to_tsvector(array_to_string(e.authors, ' ')), 'A') ||
     setweight(to_tsvector(array_to_string(e.tags, ' ')), 'B') ||
     setweight(to_tsvector(e.title), 'D') ||
     setweight(to_tsvector(e.description), 'D') AS ep_document
     FROM entrypoints e;
 
 
+CREATE MATERIALIZED VIEW IF NOT EXISTS modal_function_documents AS
+    SELECT mf.id,
+    setweight(to_tsvector(array_to_string(mf.authors, ' ')), 'A') ||
+    setweight(to_tsvector(array_to_string(mf.tags, ' ')), 'A') ||
+    setweight(to_tsvector(mf.title), 'D') ||
+    setweight(to_tsvector(mf.description), 'D') AS mf_document
+    FROM modal_functions mf;
+
+
 CREATE INDEX IF NOT EXISTS garden_documents_index ON garden_documents USING GIN(garden_document);
 CREATE INDEX IF NOT EXISTS entrypoint_documents_index ON entrypoint_documents USING GIN(ep_document);
+CREATE INDEX IF NOT EXISTS modal_function_documents_index ON modal_function_documents USING GIN(mf_document);
 
 
 CREATE OR REPLACE FUNCTION refresh_garden_documents()
-RETURNS TRIGGER AS $$
+RETURNS TRIGGER
+AS $$
 BEGIN
     REFRESH MATERIALIZED VIEW garden_documents;
     RETURN NEW;
 END;
-$$ LANGUAGE plpgsql;
+$$
+LANGUAGE plpgsql
+;
 
 
 CREATE OR REPLACE FUNCTION refresh_entrypoint_documents()
-RETURNS TRIGGER AS $$
+RETURNS TRIGGER
+AS $$
 BEGIN
     REFRESH MATERIALIZED VIEW entrypoint_documents;
     RETURN NEW;
 END;
-$$ LANGUAGE plpgsql;
+$$
+LANGUAGE plpgsql
+;
+
 
+CREATE OR REPLACE FUNCTION refresh_modal_function_documents()
+RETURNS TRIGGER
+AS $$
+BEGIN
+    REFRESH MATERIALIZED VIEW modal_function_documents;
+    RETURN NEW;
+END;
+$$
+LANGUAGE plpgsql
+;
```

Comment:
Just curious, is there some plpgsql linter running now that wasn't before?

Reply:
I added a [SQL formatter](https://pypi.org/project/sql-formatter/) to my local dev setup. I'm not sold on adding it to the dev dependencies yet.